tests/queries/0_stateless/03547_reinterpret_to_array.sql

```diff
@@ -0,0 +1,48 @@
+-- Tests for reinterpret(<source string>, <array_of_fixed_type>)
+
+-- Verify correct destination type is instantiated
```

Comment:
Testing in ClickHouse has the general problem that it is hard to map queries (.sql file) and expected results (.reference file) during development and if something goes wrong. This can be mitigated by turning comments in the sql file into SELECTs, e.g.

```sql
-- Verify correct destination type is instantiated
```

-->

```sql
SELECT 'Verify correct destination type is instantiated';
```

These will become "headlines" in the reference file and improve readability.